apiserver/facades/client/action/action.go

```diff
@@ -113,7 +117,11 @@ func (a *ActionAPI) FindActionTagsByPrefix(arg params.FindTags) (params.FindTags
 
 	response := params.FindTagsResults{Matches: make(map[string][]params.Entity)}
 	for _, prefix := range arg.Prefixes {
-		found := a.state.FindActionTagsByPrefix(prefix)
+		m, err := a.state.Model()
+		if err != nil {
+			return params.FindTagsResults{}, errors.Trace(err)
+		}
```

Comment:
It might be best to grab the Model outside of the loop body to avoid unnecessary work